app/uk/gov/hmrc/gform/sharedmodel/TaxPeriods.scala

```diff
@@ -73,23 +72,23 @@ final case class RetrievedObligations(obligation: NonEmptyList[TaxResponse]) ext
 
 object Obligations {
   import JsonUtils._
-  implicit val format: OFormat[Obligations] = derived.oformat[Obligations]
+  implicit val format: OFormat[Obligations] = derived.oformat()
 }
 
 case class IdNumberValue(value: String) extends AnyVal
 
 object IdNumberValue {
-  implicit val format: OFormat[IdNumberValue] = derived.oformat
+  implicit val format: OFormat[IdNumberValue] = derived.oformat()
 }
 
 case class RecalculatedTaxPeriodKey(fcId: FormComponentId, hmrcTaxPeriod: HmrcTaxPeriod)
 object RecalculatedTaxPeriodKey {
-  implicit val format: OFormat[RecalculatedTaxPeriodKey] = derived.oformat
+  implicit val format: OFormat[RecalculatedTaxPeriodKey] = derived.oformat()
 }
 case class HmrcTaxPeriodWithEvaluatedId(
   recalculatedTaxPeriodKey: RecalculatedTaxPeriodKey,
   idNumberValue: IdNumberValue)
 
 object HmrcTaxPeriodWithEvaluatedId extends JsonUtils {
-  implicit val format: OFormat[HmrcTaxPeriodWithEvaluatedId] = derived.oformat
+  implicit lazy val format: OFormat[HmrcTaxPeriodWithEvaluatedId] = derived.oformat()
```

Comment:
Why is `lazy` here?

Reply:
I was trying something, its not required. Removing it

Reply:
Removed